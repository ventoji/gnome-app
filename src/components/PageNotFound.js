import React from 'react';
import { Link } from 'react-router-dom';

const PageNotFound = () => {
    return (
        <div className="not-found">
            <p> It seems the page you are trying to access doesn't exits</p>
            <Link to={`${process.env.SUBDIRECTORY ? process.env.SUBDIRECTORY  : '/'}`} > Please come back home </Link> 
        </div>
    );
};

export default PageNotFound;