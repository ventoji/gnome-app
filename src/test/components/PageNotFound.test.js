import React from 'react';
import {shallow} from 'enzyme';
import PageNotFound from '../../components/PageNotFound';

test('should correctly render PageNotFound ', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();

});