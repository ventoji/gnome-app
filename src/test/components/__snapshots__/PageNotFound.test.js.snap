// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render PageNotFound  1`] = `
<div
  className="not-found"
>
  <p>
     It seems the page you are trying to access doesn't exits
  </p>
  <Link
    to="/"
  >
     Please come back home 
  </Link>
</div>
`;
